### PR TITLE
Use core-api via NGINX

### PR DESCRIPTION
**Pull request by teogeb**
Using `http://127.0.0.1` instead of `http://127.0.0.1:8081/streamr-core`

Note that STREAMR_URL is overwritten here: https://github.com/streamr-dev/streamr-docker-dev/blob/master/docker-compose.yml#L98-L121

### CODE REVIEW

File: configs/docker-1.env.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
     "name": "docker-1",
     "hostname": "0.0.0.0",
     "port": 30315,
-    "advertisedWsUrl": "ws://10.200.10.1:30315",
+    "advertisedWsUrl": "ws://127.0.0.1:30315",
```
Comment (harbu) [inline configs/docker-1.env.json:7]: Ummm not sure about this change, this should stay as is most likely.

File: configs/docker-1.env.json [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
     "isStorageNode": true,
     "trackers": {
       "registryAddress": "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF",
-      "jsonRpcProvider": "http://10.200.10.1:8545"
+      "jsonRpcProvider": "http://127.0.0.1:8545"
```
Comment (harbu) [inline configs/docker-1.env.json:11]: Ummm not sure about this change, this should stay as is most likely.

File: configs/docker-2.env.json [hunk 3]
```diff
@@ -4,11 +4,11 @@
     "name": "docker-2",
     "hostname": "0.0.0.0",
     "port": 30316,
-    "advertisedWsUrl": "ws://10.200.10.1:30316",
+    "advertisedWsUrl": "ws://127.0.0.1:30316",
```
Comment (harbu) [inline configs/docker-2.env.json:7]: Ummm not sure about this change, this should stay as is most likely.

File: configs/docker-2.env.json [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
     "isStorageNode": false,
     "trackers": {
       "registryAddress": "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF",
-      "jsonRpcProvider": "http://10.200.10.1:8545"
+      "jsonRpcProvider": "http://127.0.0.1:8545"
```
Comment (harbu) [inline configs/docker-2.env.json:11]: Ummm not sure about this change, this should stay as is most likely.

File: configs/docker-3.env.json [hunk 5]
```diff
@@ -4,11 +4,11 @@
     "name": "docker-3",
     "hostname": "0.0.0.0",
     "port": 30317,
-    "advertisedWsUrl": "ws://10.200.10.1:30317",
+    "advertisedWsUrl": "ws://127.0.0.1:30317",
```
Comment (harbu) [inline configs/docker-3.env.json:7]: Ummm not sure about this change, this should stay as is most likely.

File: configs/docker-3.env.json [hunk 6]
```diff
@@ -4,11 +4,11 @@
...
     "isStorageNode": false,
     "trackers": {
       "registryAddress": "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF",
-      "jsonRpcProvider": "http://10.200.10.1:8545"
+      "jsonRpcProvider": "http://127.0.0.1:8545"
```
Comment (harbu) [inline configs/docker-3.env.json:11]: Ummm not sure about this change, this should stay as is most likely.

**Review by harbu — CHANGES_REQUESTED** [review]

**Review by harbu — APPROVED** [review]

**Review by kare — APPROVED** [review]
LGTM

**Pull request merged**